### DISCUSSION TITLE
Send notifications from view

### DISCUSSION THREAD

**Discussion by studiowizjo** [filamentphp/filament#3582] [Help] [answered]
How  notifications can be send from view? I tried approach like below, but without success.

```js
x-data="{
    copyUrlToClipboard() {
        navigator.clipboard.writeText('{{ $comment->scrollUrl() }}');

        $dispatch('notify.window', { //or just: 'notify'
            message: 'URL was copied to clipboard',
            status: 'success',
        });
    }
}"
```

**Comment by danharrin** [comment]
It needs to be sent from a Livewire component class

**Reply by dinwwwh** [reply]
Is this possible in the future? Currently, the community has a ton of `js toast message` packages. So I think, I can use `filament notifications` instead of that packages in client/js.

**Reply by danharrin** [reply]
I've opened an issue to track the possible development of this feature - #4239

I'll see if it's possible or not

**Comment by danharrin** [comment, marked as answer]
Implemented in #4444

**Reply by studiowizjo** [reply]
I have just updated to newest version, made new build and replace above code with:

```js
x-data="{
    copyUrlToClipboard() {
        navigator.clipboard.writeText('{{ $task->url() }}');

        new window.Notification()
            .title('Copied!')
            .icon('heroicon-o-document-text') // [tl! focus:start]
            .iconColor('success') // [tl! focus:end]
            .send();
    }
}"
```

and console shows error: `Uncaught TypeError: Failed to construct 'Notification': 1 argument required, but only 0 present.`. Invoking with argument: `new window.Notification('my_uuid')` changes error to: `Uncaught TypeError: (intermediate value).title is not a function`.

**Reply by danharrin** [reply]
I wonder if you have a different package that is registering a window.Notification class too?

In your app.js compiled script you could import the `Notification` class and register it to `window.FilamentNotification` if you wanted to :) Check the bottom of the docs to see how to import
